.github/workflows/build.yml

```diff
@@ -37,12 +37,39 @@ jobs:
       - uses: ./.github/actions/list-variants
         id: get-variants
 
-  build:
+  package-prep:
     needs: list-variants
     runs-on:
       group: bottlerocket
       labels: bottlerocket_ubuntu-latest_32-core
     continue-on-error: true
+    strategy:
+      matrix:
+        arch: [x86_64, aarch64]
+      fail-fast: false
+    name: "Prebuild ${{ matrix.arch }} packages"
+    steps:
+      - uses: actions/checkout@v4
+      - name: Preflight step to set up the runner
+        uses: ./.github/actions/setup-node
+      # Build some of the most common packages so later builds pick them up from the cache and don't rebuild
+      # TODO: Make some of this dynamic like we do with discovering variants
+      - run: |
+          cargo make -e BUILDSYS_ARCH=${{ matrix.arch }} -e PACKAGE=kernel-5_10 build-package
+        shell: bash
+      - run: |
+          cargo make -e BUILDSYS_ARCH=${{ matrix.arch }} -e PACKAGE=kernel-5_15 build-package
+        shell: bash
+      - run: |
+          cargo make -e BUILDSYS_ARCH=${{ matrix.arch }} -e PACKAGE=kernel-6_1 build-package
+        shell: bash
```

Comment:
These are variant-specific and need to be rebuilt for vmware and metal, currently.